docs/generate-time-series/go.rst

```diff
@@ -0,0 +1,374 @@
+
+.. _gen-ts-go:
+
+==================================
+Generate time series data using Go
+==================================
+
+This tutorial will show you how to generate some :ref:`experimental time 
+series data <gen-ts>` from information about the `International Space Station`_
+using `Go`_.
+
+.. SEEALSO::
+
+    :ref:`gen-ts`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Prerequisites
+=============
+
+You must have CrateDB :ref:`installed and running <install-run>`.
+
+Make sure you're running an up-to-date version of `Go`_. We recommend 
+Go 1.11 or higher since we will be making use of `modules`_. 
+
+Most of this tutorial is designed to be run as a local Go project using Go 
+tooling since the `compilation`_ unit is the package and not a single line.
+
+To begin, create a project directory and navigate into it:
+
+.. code-block:: console
+
+    $ mkdir time-series-go
+    $ cd time-series-go
+
+Next, choose a module path and create a `go.mod` file that declares it:
+
+.. code-block:: console
+
+    $ go mod init example.com/time-series-go
+
+You should see a `go.mod` file in the current directory with contents similar to:
+
+.. code-block:: console
+
+    module example.com/time-series-go
+
+    go 1.14
+
+Next, create a file named `main.go` in the same directory:
+
+.. code-block:: console
+
+   $ touch main.go
+
+Open up your favorite code editor. We will be editing this file a lot!
+
+
+Get ISS telemetry data
+======================
+
+You can get telemetry data from `Open Notify`_, a third-party service that
+provides a simple API to consume data from NASA (specifically, the current
+location of the International Space Station). The endpoint for this data is
+`<http://api.open-notify.org/iss-now.json>`_.
+
+
+In your `main.go` file, declare the main package at the top (to tell the 
+compiler that our program is an executable), import some packages from the 
+`standard library`_ that will be used in this tutorial and declare a main 
+function which will be the entry point of our executable program:
+
+.. code-block:: js
+
+    package main
+
+    import (
+        "encoding/json"
+        "fmt"
+        "io/ioutil"
+        "log"
+        "net/http"
+    )
+
+    func main() {
+
+    }
+
+Then, look at the JSON data that gets returned from going to the Open
+Notify API endpoint at `<http://api.open-notify.org/iss-now.json>`_. The 
```

Comment:
do you want the reader to open this in the browser? or would it be better done on the command line? I think we should be explicit, either way :)